cudatoolkit-dev/scripts/cuda_activate.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash 
 
-export PATH_BACKUP=$PATH
-export CUDA_HOME_BACKUP=$CUDA_HOME
-export CUDA_HOME=$CONDA_PREFIX/pkgs/cuda
-export PATH=$CUDA_HOME/bin:$PATH
\ No newline at end of file
+cd $CONDA_PREFIX/bin
+find . -type l -exec unlink {} \;
```

Comment:
@scopatz, Let me know if there is a better way of dealing with existing soft links. For now, I am deleting the existing soft links before creating new ones at `conda activate env` time.